modular_skyrat/modules/cyborg/code/robot_upgrades.dm

```diff
@@ -215,3 +215,33 @@
 		qdel(plasteel_energy)
 	for(var/datum/robot_energy_storage/titanium/titanium_energy in borgo.model.storages)
 		qdel(titanium_energy)
+
+// funny borg inducer upgrade
+/obj/item/borg/upgrade/inducer//SKYRAT EDIT - ICON OVERRIDEN BY AESTHETICS - SEE MODULE
+	name = "engineering cyborg inducer upgrade"
+	desc = "An inducer device for the engineering cyborg."
+	//icon = 'icons/obj/storage.dmi'
```

Comment:
```suggestion
```
Don't leave code commented out also uhhh this probably needs an icon. But I can't really check on my phone.